### PR TITLE
Renew PrestaShop UI Kit branding 

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Renew PrestaShop UI Kit branding, agreed by @matks
| Type?             | new feature
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | See https://github.com/PrestaShop/PrestaShop/issues/36615
| Sponsor company   | @PrestaShopCorp
| How to test?      |

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hello @tblivet!

This is your first pull request on [prestashop-ui-kit](https://github.com/PrestaShop/prestashop-ui-kit/) repository of the PrestaShop project.

Thank you, and welcome to this Open Source community!

File: scss/_imports.scss
Comment (jolelievre) [inline scss/_imports.scss:13]: Isn't it overkill to have a different font file for each value of boldness?

File: scss/_root.scss
Comment (jolelievre) [inline scss/_root.scss:1]: I'm curious to understand the logic behind these variables, they don't seem to rely on some semantic meanings but rather on their values so I don't understand the advantage of using them instead of hard coded values

We also must think about the future developers that are going to contribute to the core, how will they understand the logic behind these variables? If it's not developer friendly changes are they won't be used, the reviewers also won't detect the inconsistencies and we could soon end up with some mess in the CSS again

File: scss/_root.scss
Comment (jolelievre) [inline scss/_root.scss:74]: These ones especially I don't understand the advantage

File: scss/_root.scss
Comment (jolelievre) [inline scss/_root.scss:15]: What is the meaning behind these suffix numbers?

File: scss/_variables.scss
Comment (jolelievre) [inline scss/_variables.scss:1]: Ah great this means the old variables are still usable right? But their values are now based on the new ones?

**Review by jolelievre — COMMENTED** [review]
Great job @tblivet 

I just have a few questions to understand the new convention in variables I also think this should be documented so that developers understand the logic behind it If it's not documented and explained I'm afraid people won't use the new convention and all the huge cleaning job you did in the core will be wasted as time goes on

File: scss/_imports.scss
Comment (jolelievre) [reply scss/_imports.scss:13]: Can you add the `font-display` setting to lazy load the fonts?

**Review by jolelievre — APPROVED** [review]
Thanks @tblivet

**Review by mattgoud — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
